_includes/menu-item-ids/advtable-menu-items.md

```diff
@@ -1,3 +1,4 @@
-| Menu item identifier | [Default Menu Location]({{site.baseurl}}/configure/editor-appearance/#examplethetinymcedefaultmenuitems) | Description                                            |
-| -------------------- | -------------------------------------------------------------------------------------------------------- | ------------------------------------------------------ |
-| `advtablesort`       | Table                                                                                                    | Advanced tables sort menu item with related controls.  |
\ No newline at end of file
+| Menu item identifier    | [Default Menu Location]({{site.baseurl}}/configure/editor-appearance/#examplethetinymcedefaultmenuitems) | Description                                                                             |
+| ----------------------- | -------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+| `advtablesort`          | Table                                                                                                    | Advanced tables sort menu item with related controls.                                   |
+| `advtablerownumbering`  | _Not Applicable_                                                                                         | Adds and removes a row numbering column on the selected table. {{site.requires_5_9v}} |
```

Comment:
Should we change "not applicable" to "Not included in a menu by default"? Or is `_Not Applicable_` a pattern from elsewhere in the docs